src/docfx/lib/markdown/MarkdigUtility.cs

```diff
@@ -266,7 +292,7 @@ public static bool IsVisible(this MarkdownObject markdownObject)
             {
                 var nodeVisible = node switch
                 {
-                    HtmlBlock htmlBlock => htmlBlock.Lines.Lines.Any(line => HtmlUtility.IsVisible(line.Slice.ToString())),
+                    HtmlBlock htmlBlock => HtmlUtility.IsVisible(string.Join("", htmlBlock.Lines.Lines)),
```

Comment:
nit: Join by `''` eats new lines:
```suggestion
                    HtmlBlock htmlBlock => HtmlUtility.IsVisible(htmlBlock.Lines.ToString()),
```

Reply:
ok